component/vshn_postgres.jsonnet

```diff
@@ -0,0 +1,232 @@
+local com = import 'lib/commodore.libjsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+
+local comp = import 'lib/appcat-compositions.libsonnet';
+local crossplane = import 'lib/crossplane.libsonnet';
+
+local common = import 'common.libsonnet';
+local xrds = import 'xrds.libsonnet';
+
+local inv = kap.inventory();
+local params = inv.parameters.appcat;
+local pgParams = params.services.vshn.postgres;
+
+local defaultDB = 'postgres';
+local defaultUser = 'postgres';
+local defaultPort = '5432';
+
+
+local connectionSecretKeys = [
+  'POSTGRESQL_URL',
+  'POSTGRESQL_DB',
+  'POSTGRESQL_HOST',
+  'POSTGRESQL_PORT',
+  'POSTGRESQL_USER',
+  'POSTGRESQL_PASSWORD',
+  'ca.crt',
+];
+
+local xrd = xrds.XRDFromCRD(
+  'xvshnpostgresqls.appcat.vshn.io',
```

Comment:
Did we define if the api group should be `appcat.vshn.io` or `vshn.appcat.vshn.io`?

Reply:
It's `vshn.appcat.vshn.io`. Updated the code.